### ISSUE TITLE
Uncaught TypeError: item.on is not a function

### ISSUE THREAD

**Issue by andyngo** [t9md/atom-choose-pane#9]
Hi, just to bring this to your attention. Latest Atom beta might introduce some issues to Choose Pane.

Steps to reproduce:

1. Run the latest Atom beta (1.16.0-beta0)
2. Toggle tree view
3. Error notification pops up

![atom-beta-choose-pane-bug](https://cloud.githubusercontent.com/assets/278293/23777134/80166df0-056d-11e7-8ccc-cd2e45954fef.gif)

<img width="390" alt="screenshot 2017-03-10 08 42 08" src="https://cloud.githubusercontent.com/assets/278293/23777164/a47968dc-056d-11e7-82fb-9280dc590300.png">

**Atom**: 1.16.0-beta0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [choose-pane](https://github.com/t9md/atom-choose-pane) package 0.6.0

### Stack Trace

Uncaught TypeError: item.on is not a function

```
At /Users/andyngo/.atom/packages/choose-pane/lib/main.coffee:67

TypeError: item.on is not a function
    at /packages/choose-pane/lib/main.coffee:67:14
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at PanelContainer.addPanel (/app.asar/src/panel-container.js:56:18)
    at Workspace.addPanel (/app.asar/src/workspace.js:1181:43)
    at Workspace.addLeftPanel (/app.asar/src/workspace.js:1054:17)
    at TreeView.module.exports.TreeView.attach (/app.asar/node_modules/tree-view/lib/tree-view.js:405:29)
    at TreeView.module.exports.TreeView.show (/app.asar/node_modules/tree-view/lib/tree-view.js:380:12)
    at TreeView.module.exports.TreeView.toggle (/app.asar/node_modules/tree-view/lib/tree-view.js:375:21)
    at /app.asar/node_modules/tree-view/lib/main.js:34:39)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
    at /app.asar/src/command-registry.js:3:59
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:599:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:390:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/app.asar/src/window-event-handler.js:106:36)
    at /app.asar/src/window-event-handler.js:3:59)
```

### Commands

```
     -2:05.4.0 choose-pane:start (input.hidden-input)
  3x -2:05.4.0 blur (input.hidden-input)
     -2:05.2.0 choose-pane:last-focused (input.hidden-input)
  2x -2:05.2.0 blur (input.hidden-input)
     -2:04.9.0 choose-pane:start (input.hidden-input)
  3x -2:04.8.0 blur (input.hidden-input)
     -2:01.7.0 choose-pane:last-focused (input.hidden-input)
  2x -2:01.7.0 blur (input.hidden-input)
     -2:01.4.0 choose-pane:start (input.hidden-input)
  3x -2:01.4.0 blur (input.hidden-input)
     -2:01.0 choose-pane:last-focused (input.hidden-input)
  2x -2:01.0 blur (input.hidden-input)
     -1:54.3.0 choose-pane:start (input.hidden-input)
 18x -1:54.3.0 blur (input.hidden-input)
     -0:20.2.0 tree-view:toggle (input.hidden-input)
  2x -0:00.1.0 blur (input.hidden-input)
```

### Non-Core Packages

```
aligner 0.22.3 
ariake-dark-syntax 0.1.3 
choose-pane 0.6.0 
emmet 2.4.3 
file-type-icons 1.3.3 
gloom 0.4.7 
indent-guide-improved 1.4.12 
language-babel 2.56.2 
language-slim 0.5.0 
language-vue 0.21.2 
lines 0.13.1 
narrow 0.37.1 
octocat-syntax 1.3.1 
pigments 0.39.0 
prettier-atom 0.23.0 
project-manager 3.3.3 
relative-numbers 0.7.1 
ruby-slim 0.2.0 
sync-settings 0.8.1 
synced-sidebar 0.4.3 
todo-show 1.11.0 
tree-view-git-status 1.3.1 
vim-mode-plus 0.83.0 
vim-mode-plus-keymaps-for-surround 0.2.1 
```

**Comment by t9md** [comment]
Thanks.
This is because of removal of jQuery from tree-view.
The error throwing part is for tracking last-focused state for tree-view panel.
After I struggling to revive this feature on atom-beta1.16.0, I couldn't find out the way.
I don't know why 'focused' event on panelContainer element is not fired.

So what I did is just disable this feature for version 1.16.0 above.
So what was changed is "tree-view focus is not counted as last-focus from atom version 1.16.0".

**Comment by andyngo** [comment]
Thank you for patching this so quickly! Yeah, removing tree-view last-focus is not a big deal since choose-pane already does a great job at switching panes very quickly.

**Comment by t9md** [comment]
Not released yet, I found the way, wait for a minutes.

**Comment by t9md** [comment]
Released v0.6.1.
Ignore above my comment for remove-feature-on-v1.16.0-beta.
Now atom-v1.16.0-beta should works in same way as in atom-v1.15.0.
